server.ts

```diff
@@ -50,6 +50,7 @@ import { AdminAvailableStudents } from './src/db_queries/admin_available_student
 import { AdminPartnersMentors } from './src/db_queries/admin_partners_mentors';
 import { AdminPartnersMentorStats } from './src/db_queries/admin_partners_mentors_stats';
 import { AdminPartnersProjects } from './src/db_queries/admin_partners_projects'
+import { adminPartnersStudents } from './src/db_queries/admin_partners_students'
```

Comment:
We should have { AdminPartnersStudents } instead of { adminPartnersStudents } here.